### PR TITLE
Fix 1779

### PR DESCRIPTION
**Pull request by saroufimc1**
Addressing issue #1779. Correct the assignment of model.wv.syn0_ngrams after trimming of unused ngrams (model.wv.syn0_ngrams.shape[0] <= self.bucket). 

It is 2x slower on pre-trained Wikipedia models from Facebook's fastText than if we do not trim unused ngrams. Therefore, I recommend dealing with #1261. 

For the test case, we need an actual small Facebook FastText model saved in bin format and make sure that we have model.wv.syn0_ngrams.shape[0] <= self.bucket.

Related to case #1787.

### CODE REVIEW

File: gensim/models/wrappers/fasttext.py [hunk 1]
```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 #
-# Author: Jayant Jain <jayantjain1992@gmail.com>
```
Comment (menshikh-iv) [inline gensim/models/wrappers/fasttext.py]: Why you remove this?

**Comment by menshikh-iv** [comment]
@saroufimc1 #1777 merged, please resolve merge conflict first

**Comment by menshikh-iv** [comment]
Ping @saroufimc1, when you plan to finish this PR?

**Comment by menshikh-iv** [comment]
Unfortunately, you merged conflict incorrectly (look at current state of `gensim/models/wrappers/fasttext.py` from `develop`), your code edition should be in https://github.com/RaRe-Technologies/gensim/blob/develop/gensim/models/deprecated/fasttext_wrapper.py

Are you planning to finish PR?

**Comment by saroufimc1** [comment]
@menshikh-iv  Sorry about that, I am not familiar with all the changes of the source code that happened in the meantime. Is it possible that you take my code and do the merging yourself since you are more familiar with the changes? It would be much faster I guess.

File: gensim/models/wrappers/fasttext.py (hunk 1 above)
Comment (saroufimc1) [reply gensim/models/wrappers/fasttext.py]: Wasn't sure about the header. Feel free to take the code and do whatever you want with it.

**Comment by menshikh-iv** [comment]
@saroufimc1 no problem, I close this with "almost complete" label, we will return to this later and fix it, thanks!

CC: @manneshiva

**Pull request closed without merging**